README.rst

```diff
@@ -335,6 +335,23 @@ The following flags are accepted:
 | Determines the proto option Gazelle uses to group .proto files into rules                        |
 | when in ``package`` mode. See details in `Directives`_ below.                                    |
 +--------------------------------------------------------------+-----------------------------------+
+| :flag:`-proto_strip_import_prefix example.com/repo`          |                                   |
++--------------------------------------------------------------+-----------------------------------+
+| The prefix to strip from the paths of the .proto files in the proto_library rule.                |
```

Comment:
In the text here and for the attribute below, just say something like "Sets the `import_prefix` attribute of generated `proto_library` rules. This is a prefix to add to import paths of .proto files.". Make sure to link `import_prefix` to `https://docs.bazel.build/versions/master/be/protocol-buffer.html#proto_library.import_prefix`. No need to give much more detail than that.